tests/test_views.py

```diff
@@ -108,9 +114,15 @@ def test_existing_notification(self):
     # XXX: Add tests for POST notifications
 
 
+@utils.override_settings(ROOT_URLCONF='tests.test_urls')  # Django<=1.11.11
```

Comment:
I think we can just replace `ROOT_URLCONF` with `tests.test_urls` in `tests/settings.py`.

It's simpler and I don't _think_ it breaks anything.

Reply:
ah! ok!

The simple `ROOT_URLCONF` throws a namespace error

Reply:
Yup. But proper usage of the app requires URLs like the ones you requested in `test_urls`, so it makes sense to make those stick.